### PR TITLE
Build all assets and a release with drone

### PR DESCRIPTION
**Pull request by metalmatze**
Pretty straight forward and `.drone.yml` inspired by the one from github.com/go-gitea/gitea. 😄

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -2,6 +2,23 @@ workspace:
...
+      - make install
+      - make build
+      - make release
+
+  release:
+    image: plugins/s3
+    path_style: true
+    strip_prefix: dist/
+    source: dist/*
+    target: /theme/master
```
Comment (tboerger) [inline .drone.yml]: Just /theme

File: .drone.yml [hunk 2]
```diff
@@ -2,6 +2,23 @@ workspace:
...
+    commands:
+      - make install
+      - make build
+      - make release
+
+  release:
+    image: plugins/s3
+    path_style: true
+    strip_prefix: dist/
+    source: dist/*
```
Comment (tboerger) [inline .drone.yml]: You want to create an tar.gz archive and upload only that

**Review by tboerger — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 2 above)
Comment (metalmatze) [reply .drone.yml]: I'll do that right after I get home! 😊

**Comment by metalmatze** [comment]
Please squash when merging! 😉

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,4 @@
+/dist/
```
Comment (tboerger) [inline .gitignore]: This is not used anymore

File: .drone.yml [hunk 4]
```diff
@@ -2,6 +2,22 @@ workspace:
   path: src
 
 pipeline:
+  build:
+    image: node
```
Comment (tboerger) [inline .drone.yml]: Let's use webhippie/node:latest like we do for golang :)

Comment (metalmatze) [reply .drone.yml]: https://hub.docker.com/r/webhippie/node/ returns 404? :(

Comment (metalmatze) [reply .drone.yml]: Furthermore I don't really see much benefit just for the building pipeline. We're not running our app with node. Probably the official one will be updated more frequently? 😊

Comment (tboerger) [reply .drone.yml]: Oh it's nodejs

**Comment by tboerger** [comment]
LGTM

**Review by tboerger — APPROVED** [review]

**Comment by Bwko** [comment]
LGTM

**Pull request merged**